seatunnel-connectors-v2/connector-cdc/connector-cdc-mysql/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/source/MySqlIncrementalSourceFactory.java

```diff
@@ -99,9 +108,42 @@ public Class<? extends SeaTunnelSource> getSourceClass() {
             TableSource<T, SplitT, StateT> restoreSource(
                     TableSourceFactoryContext context, List<CatalogTable> restoreTables) {
         return () -> {
+            ReadonlyConfig config = context.getOptions();
+            Optional<String> optional = config.getOptional(CatalogOptions.TABLE_PATTERN);
```

Comment:
Should this parameter restrict the coexistence of table names and database names simultaneously?

Reply:
It is very necessary in the scenario of sub-database and sub-table.

Reply:
I'm sorry for the misunderstanding in my previous responses，Should it also conflict with the DATABASE_NAMES parameter?

Reply:
I see . let me verify